index.js

```diff
@@ -1,30 +1,32 @@
-const IncomingWebhook = require('@slack/client').IncomingWebhook;
+const { IncomingWebhook } = require('@slack/client');
 const humanizeDuration = require('humanize-duration');
+const Octokit = require('@octokit/rest');
 const config = require('./config.json');
 
 module.exports.webhook = new IncomingWebhook(config.SLACK_WEBHOOK_URL);
 module.exports.status = config.GC_SLACK_STATUS;
 
 // subscribe is the main function called by GCF.
-module.exports.subscribe = (event, callback) => {
-  const build = module.exports.eventToBuild(event.data.data);
+module.exports.subscribe = async (event) => {
```

Comment:
With async, should the code wrapped in a try,catch so that we can return errors? 
Can you write a test in which something errors out?

Reply:
Try catch added and test written, please review!